templates/onap/config.yml

```diff
@@ -0,0 +1,5 @@
+name: ONAP	
+description: |
+  ONAP is a comprehensive platform for real-time, policy-driven orchestration and automation of physical and virtual network functions that will enable software, network, IT and cloud providers and developers to rapidly create new services.
+version: 1.1
```

Comment:
Version value here has to match with `templates/onap/0/rancher-compose.yml` version value.